mkosi/resources/__init__.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: PSF-2.0
+# Based on code from https://github.com/python/cpython/blob/main/Lib/importlib/resources/_common.py
+
+import contextlib
+import functools
+import os
+import stat
+import tempfile
+from collections.abc import Iterator
+from contextlib import AbstractContextManager
+from importlib.abc import Traversable
```

Comment:
Since it does complain, this could be made, since we probably will have to add this anyway once 3.14 is released.
```python
import sys
if sys.version_info < (3, 13):
    from importlib.abc import Traversable
else:
    from importlib.resources.abc import Traversable
```